docs/source/tutorial-rb.rst

```diff
@@ -0,0 +1,789 @@
+inner product on the target Hilbert-space. Let's check that:
+
+.. jupyter-execute::
+
+    np.linalg.cond(pod_basis.gramian(fom.h1_0_semi_product))
+
+Now, let us compare how the POD performs against the greedy algorithm in the worst-case
+best-approximation error:
+
+.. jupyter-execute::
+
+    pod_errors = compute_proj_errors_orth_basis(pod_basis, V, fom.h1_0_semi_product)
+    
+    plt.figure()
+    plt.semilogy(trivial_errors, label='trivial')
+    plt.semilogy(greedy_errors, label='greedy')
+    plt.semilogy(pod_errors, label='POD')
+    plt.ylim(1e-1, 1e1)
+    plt.legend()
+    plt.show()
+
+As it turns out, the POD spaces perform even slightly better than the greedy spaces.
+Why is that the case? Note that for finite training or validation sets, both considered error measures
+are equivalent. In particular:
+
+.. math:: 
+    \sup_{k = 1,\ldots,K} \inf_{v \in V_N} \|u(\mu_k) - v\| \leq
+    \left[\sum_{k = 1}^K \inf_{v \in V_N} \|u(\mu_k) - v\|^2\right]^{1/2} \leq
+    \sqrt{K} \cdot \sup_{k = 1,\ldots,K} \inf_{v \in V_N} \|u(\mu_k) - v\|.
+
+Since POD spaces are optimal in the :math:`\ell^2`-error, they miss the error of the
+optimal space in the Kolmogorov sense by at most a factor of :math:`\sqrt{K}`, which in
+our case is :math:`5`. On the other hand, the greedy algorithm also only produces
+quasi-optimal spaces. -- For very large training sets with more complex parameter dependence,
+differences between both spaces may be more significant.
+
+Finally, it is often insightful to look at the POD modes themselves:
+
+.. jupyter-execute::
+
+    fom.visualize(pod_basis)
+
+As you can see, the first (more important) basis vectors account for the approximation of
+the solutions in the bulk of the subdomains, whereas the higher modes are responsible for
+approximating the solutions at the subdomain interfaces.
+
+
+.. _weakgreedy:
+
+Weak greedy algorithm
+---------------------
+
+Both POD and the strong greedy algorithm require the computation of all 
+:meth:`solutions <pymor.models.interface.Model.solve>` :math:`u(\mu)`
+for all |parameter values| :math:`\mu` in the `training_set`. So it is
+clear right from the start that we cannot afford very large training sets.
+Otherwise we would not be interested in model order reduction
+in the first place. This is a problem when the number of |parameters|
+increases and/or the solution depends less uniformly on the |parameters|.
+
+Reduced basis methods have a very elegant solution to this problem, which
+allows training sets that are orders of magnitude larger than the trainings
+sets affordable for POD: instead of computing the best-approximation error
+we only compute surrogate
+
+.. math:: \inf_{v \in V_N} \|u(\mu) - v\| \approx \mathcal{E}(\mu)
+
+for it. Replacing the best-approximation error by this surrogate in the :ref:`stronggreedy`
+algorithm, we arrive at the :meth:`weak greedy <pymor.algorithms.greedy.weak_greedy>`
```

Comment:
Right now,
```
:ref:`stronggreedy` algorithm
```
is converted to "Strong greedy algorithm algorithm".

Reply:
Fixed.